src/Microsoft.AspNet.Mvc.ViewFeatures/Logging/DefaultViewComponentInvokerLoggerExtensions.cs

```diff
@@ -40,10 +40,10 @@ public static void ViewComponentExecuting(this ILogger logger, ViewComponentCont
         public static void ViewComponentExecuted(
             this ILogger logger,
             ViewComponentContext context,
-            int startTime,
+            DateTime startTime,
             object result)
         {
-            var elapsed = new TimeSpan(Environment.TickCount - startTime);
+            var elapsed = DateTime.UtcNow - startTime;
```

Comment:
We don't care about accuracy beyond the ms level. What we're doing here is intentional